modules/sops/default.nix

```diff
@@ -77,13 +80,27 @@ let
           Sops file the secret is loaded from.
         '';
       };
+      sopsFiles = mkOption {
+        type = types.listOf types.path;
+        defaultText = "\${config.sops.defaultSopsFile}";
+        description = ''
+          Sops file the secret is loaded from.
+        '';
+      };
       sopsFileHash = mkOption {
```

Comment:
We should deprecated this one than.

Reply:
Deprecate sopsFiles? Just added that this PR, do you mean deprecate sopsFile?

Reply:
We shouldn't really need both sopsFileHash and sopsFilesHash, no?

Reply:
They could be point to the same value.